components/brave_wallet_ui/components/desktop/views/accounts/accounts.tsx

```diff
@@ -54,21 +54,12 @@ export const Accounts = () => {
     }
   }, [])
 
-  const onClickAddAccount = React.useCallback((tabId: AddAccountNavTypes) => () => {
-    switch (tabId) {
-      case 'create': return history.push(WalletRoutes.CreateAccountModalStart)
-      case 'hardware': return history.push(WalletRoutes.AddHardwareAccountModalStart)
-      case 'import': return history.push(WalletRoutes.ImportAccountModalStart)
-      default: return history.push(WalletRoutes.AddAccountModal)
-    }
-  }, [])
-
   // memos
-  const primaryAccounts = React.useMemo(() => {
+  const derivedAccounts = React.useMemo(() => {
     return accounts.filter((account) => getAccountType(account) === 'Primary')
   }, [accounts])
 
-  const secondaryAccounts = React.useMemo(() => {
+  const importedAccounts = React.useMemo(() => {
     return accounts.filter((account) => getAccountType(account) === 'Secondary')
   }, [accounts])
```

Comment:
It seems filtering should be now on 
`account.accountId.kind === BraveWallet.AccountKind.kDerived`
`account.accountId.kind === BraveWallet.AccountKind.kImported`

Reply:
Works for me! Updated!